### PR TITLE
chore(book): build script + remove openssl

### PR DESCRIPTION
**Pull request by ctian1**

### CODE REVIEW

File: book/generating-proofs/basics.md [hunk 1]
```diff
@@ -13,3 +13,18 @@ To make this more concrete, let's walk through a simple example of generating a
...
+
+## Build Script
+
+If you want your program crate to be built automatically whenever you build/run your script crate, you can add a `build.rs` file inside of `script/`:
+
+```rust,noplayground
+{{#include ../../examples/fibonacci-io/script/build.rs}}
+```
+
+Make sure to also add `sp1-helper` as a build dependency in `script/Cargo.toml`:
```
Comment (puma314) [inline book/generating-proofs/basics.md:25]: should this be `sp1-core`?

File: book/getting-started/install.md [hunk 2]
```diff
@@ -3,10 +3,12 @@
 SP1 currently runs on Linux and macOS. You can either use prebuilt binaries through sp1up or
 build the toolchain and CLI from source.
 
-Make sure you have [Rust](https://www.rust-lang.org/tools/install) and OpenSSL 1.1 (`brew install openssl@1.1` or use this [guide](https://askubuntu.com/questions/1102803/how-to-upgrade-openssl-1-1-0-to-1-1-1-in-ubuntu-18-04)) installed.
+Make sure you have [Rust](https://www.rust-lang.org/tools/install) installed.
 
 ## Option 1: Prebuilt Binaries (Recommended)
 
+Currently our prebuilt binaries are built on Ubuntu 20.04 (22.04 on ARM) and macOS. If your OS is older, it's possible they won't work but you can still try.
```
Comment (puma314) [inline book/getting-started/install.md:10]: Can you say what they should do for next steps if it doesn't work?

File: book/generating-proofs/basics.md (hunk 1 above)
Comment (ctian1) [reply book/generating-proofs/basics.md:25]: no, `sp1-helper` is its own crate so that build.rs doesn't need to compile lots of stuff.

Comment (puma314) [reply book/generating-proofs/basics.md:25]: Oh, then can you make it `sp1-helper` below? it's listed as sp1-core :P

Comment (ctian1) [reply book/generating-proofs/basics.md:25]: o oops

**Review by puma314 — APPROVED** [review]

**Pull request merged**